### PR TITLE
feat(follow): Add option to follow thumbstick

### PR DESCRIPTION
**Pull request by emepetres**
When follow option is set to true, the joystick follows the pointer when it reaches its border.

This is meant to improve usability on touch screens, when the user doesn't have the feedback of getting out the joystick zone.

### CODE REVIEW

**Comment by yoannmoinet** [comment]
Hey thank you for this, I'll have a look.

File: README.md [hunk 1]
```diff
@@ -270,6 +272,9 @@ Creates square region for joystick movement
 ### `options.dynamicPage` defaults to false
 Enable if the page has dynamically visible elements such as for Vue, React, Angular or simply some CSS hiding or showing some DOM.
 
+### `options.follow` defaults to false
+Enable if the joystick must follow the thumbstick when it reaches the border.
```
Comment (yoannmoinet) [inline README.md:276]: That sounds weird, I know you wanted to follow the previous sentence, but I don't think it's working great in this context.

Maybe something like:

> Makes the joystick follow the thumbstick when it reaches the border.

WDYT?

**Review by yoannmoinet — CHANGES_REQUESTED** [review]
I've tested it, the movement is great and all, works exactly as expected.
Well done. (Even tested with the square shape, and it's working :100:)

But there seems to be an offset somewhere as soon as I reach the border of the back piece.
Not sure where it is coming from.

https://user-images.githubusercontent.com/597828/108692983-99c05a00-74fd-11eb-9ed9-007d767d45ea.mov

**Comment by emepetres** [comment]
Yep the offset is weird, in my project I set the zone to a specific div and it works as expected. Must be something css related, I'll take a look.

**Comment by yoannmoinet** [comment]
It's probably the offset of the area to which the joystick is attached too.
If the area is top left, you won't have an offset I guess.

**Comment by yoannmoinet** [comment]
@emepetres I have a fix, which also fixes other things I believe.
Do you mind if I commit in your branch?

**Comment by yoannmoinet** [comment]
I want ahead and fixed it.
Thank you very much for this addition.
It will please and fix #13 😉

**Review by yoannmoinet — APPROVED** [review]

**Comment by matthewrobb** [comment]
Thank you both!

**Comment by emepetres** [comment]
sure thanks @yoannmoinet! I couldn't find time today to fix it, glad you did :)

**Pull request merged**